wa_examples/html_wa.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "/wa/translation-alignment/QWxpZ25tZW50Q2h1bmtOb2RlOjE=",
+  "type": "Annotation",
+  "target": [
+    "urn:cts:greekLit:tlg0012.tlg001.msA:1.1-1.7",
+    {
+      "type": "SpecificResource",
+      "source": {
```

Comment:
I thought it best following from @markpatton's mockups to target the Canvas; happy to swap this out for something else if desired

Reply:
I think that is good for the moment.